testfiles/UnitaryHybridAC_DedicatedOutsideAir.idf

```diff
@@ -857,6 +857,7 @@
     0.1,                     !- Mode 9 Minimum Supply Air Mass Flow Rate Ratio
     1,                       !- Mode 9 Maximum Supply Air Mass Flow Rate Ratio
     ,                        !- Mode 10 Name
+    ,                        !- Mode 10 Supply Air Temperature Lookup Table Name
```

Comment:
@jmarrec Energy+.idd is also missing this field.
```
  A110, \field Mode 10 Name
        \type alpha
        \retaincase
        \note Enter a name for Mode 10.
        \type alpha
        \retaincase
        \note Enter the name of the Supply Air Temperature Lookup Table for Mode 10.
        \note Units for lookup table values should be in C.
        \note If this field is blank, Mode 10 will not be considered for any time step that requires ventilation, heating, cooling, humidification, or dehumidification.
  A111, \field Mode 10 Supply Air Humidity Ratio Lookup Table Name
        \type object-list
        \object-list MultivariateFunctions
        \note Enter the name of the Supply Air Humidity Ratio Lookup Table for Mode 10.
        \note Units for lookup table values should be in kgWater/kgDryAir.
        \note If this field is blank, Mode 10 will not be considered for any time step that requires ventilation, heating, cooling, humidification, or dehumidification.
  A112, \field Mode 10 System Electric Power Lookup Table Name
```

Reply:
Thanks for the catch @mjwitte 

Added IDD field in https://github.com/NREL/EnergyPlus/pull/8671/commits/670e1962dba7cc919ddaba609a4f31d6443d6e50.

This object really would be better off being an actual Extensible object... It is 5000 lines long, 660 fields.